autotest/basic.cpp

```diff
@@ -0,0 +1,70 @@
+#include <QtTest>
+
+#include "qtkeychain/keychain.h"
+
+namespace
+{
+    QByteArray generateRandomString(qsizetype size)
+    {
+        std::vector<quint32> buffer(size, 0);
+        QRandomGenerator::global()->fillRange(buffer.data(), size);
+        return QByteArray(reinterpret_cast<char *>(buffer.data()), static_cast<int>(size * sizeof(quint32))).toBase64(QByteArray::Base64UrlEncoding).mid(0, size);
```

Comment:
Jeez, I somehow thought there was a "randomString" method somewhere, but maybe that was KDE :)